src/DetailsView/reports/components/outcome-type.tsx

```diff
@@ -6,9 +6,15 @@ import { InapplicableIcon, InapplicableIconInverted } from '../../../common/icon
 import { CheckIcon, CheckIconInverted } from './../../../common/icons/check-icon';
 import { CircleIcon } from './../../../common/icons/circle-icon';
 import { CrossIcon, CrossIconInverted } from './../../../common/icons/cross-icon';
-import { InstanceOutcomeType } from './report-sections/outcome-summary-bar';
 import { RequirementOutcomeType } from './requirement-outcome-type';
 
+export type OutcomeUnits = 'percentage' | 'requirements';
+
+export type InstanceOutcomeType = 'pass' | 'fail' | 'inapplicable';
+export const InstanceOutcomeTypes: OutcomeType[] = ['fail', 'pass', 'inapplicable'];
+export const RequirementOutcomeTypes: OutcomeType[] = ['pass', 'incomplete', 'fail'];
```

Comment:
This one caught my attention:
1. can this be of type `RequirementOutcomeTypes[]` instead?
2. Also, there is a difference on where we define types for requirement and instances. While instance is all here (`InstanceOutcomeType` and `InstanceOutcomeTypes`), that's not the case for requirement (notice the import on line 9: `import { RequirementOutcomeType } from './requirement-outcome-type';`; I'm curious, why not follow the same pattern?